Assets/Scripts/Game/Entities/DaggerfallStats.cs

```diff
@@ -46,14 +46,14 @@ public partial class DaggerfallStats
 
         #region Properties
 
-        public int LiveStrength { get { return GetLiveStatValue(DFCareer.Stats.Strength); } }
-        public int LiveIntelligence { get { return GetLiveStatValue(DFCareer.Stats.Intelligence); } }
-        public int LiveWillpower { get { return GetLiveStatValue(DFCareer.Stats.Willpower); } }
-        public int LiveAgility { get { return GetLiveStatValue(DFCareer.Stats.Agility); } }
-        public int LiveEndurance { get { return GetLiveStatValue(DFCareer.Stats.Endurance); } }
-        public int LivePersonality { get { return GetLiveStatValue(DFCareer.Stats.Personality); } }
-        public int LiveSpeed { get { return GetLiveStatValue(DFCareer.Stats.Speed); } }
-        public int LiveLuck { get { return GetLiveStatValue(DFCareer.Stats.Luck); } }
+        public int LiveStrength { get { return GetLiveStatValue(DFCareer.Stats.Strength); } set { mods[0] += value - GetLiveStatValue(DFCareer.Stats.Strength); } }
```

Comment:
Please don't make any changes to "mods" array at this time. These way these values are intended to work isn't what you might expect. They are going to be flattened back to zero with every tick of effect system. Just leave them as-is for now, thank you!

Reply:
To give you more detail, I purposefully want to insulate any system from making changes here without going to through the effect system. There will be some protections built into effects to keep them in line and prevent changes "escaping" from an effect back onto live stats.